src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2001,15 +2001,15 @@ private static SymbolInfo GetSymbolInfoForSubpattern(Symbol subpatternSymbol)
         {
             if (subpatternSymbol?.OriginalDefinition is ErrorTypeSymbol originalErrorType)
             {
-                return new SymbolInfo(symbol: null, originalErrorType.CandidateSymbols.GetPublicSymbols(), originalErrorType.ResultKind.ToCandidateReason());
+                return new SymbolInfo(originalErrorType.CandidateSymbols.GetPublicSymbols(), originalErrorType.ResultKind.ToCandidateReason());
```

Comment:
moved existing callers of SymbolInfo to call into one of the other entrypoints instead of the final delegated constructor (which has been made private).